net/api/service.go

```diff
@@ -252,11 +252,13 @@ func (s *Service) AddReplicator(ctx context.Context, req *pb.AddReplicatorReques
 	if err != nil {
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
+	// This can produce an invalid Log ID, which is ok...
+	lid, _ := peer.IDFromBytes(req.LogID)
```

Comment:
We can avoid the error check here in favor of allowing invalid log ids later on down the line